test/expected/alter_hypertable.out

```diff
@@ -0,0 +1,96 @@
+\o /dev/null
+\ir include/create_single_db.sql
+SET client_min_messages = WARNING;
+DROP DATABASE IF EXISTS single;
+SET client_min_messages = NOTICE;
+CREATE DATABASE single;
+\c single
+CREATE EXTENSION IF NOT EXISTS timescaledb CASCADE;
+psql:include/create_single_db.sql:7: NOTICE:  installing required extension "postgres_fdw"
+SELECT setup_timescaledb(hostname => 'fakehost'); -- fakehost makes sure there is no network connection
+\o
+CREATE TABLE my_ht (time bigint, val integer);
+SELECT * FROM create_hypertable('my_ht', 'time');
+ create_hypertable 
+-------------------
+ 
+(1 row)
+
+ALTER TABLE my_ht ADD COLUMN val2 integer;
+\d my_ht
+     Table "public.my_ht"
+ Column |  Type   | Modifiers 
+--------+---------+-----------
+ time   | bigint  | 
+ val    | integer | 
+ val2   | integer | 
+Triggers:
+    _timescaledb_main_after_insert_trigger AFTER INSERT ON my_ht FOR EACH STATEMENT EXECUTE PROCEDURE _timescaledb_internal.main_table_after_insert_trigger()
+    _timescaledb_main_insert_trigger BEFORE INSERT ON my_ht FOR EACH ROW EXECUTE PROCEDURE _timescaledb_internal.main_table_insert_trigger()
+    _timescaledb_modify_trigger BEFORE DELETE OR UPDATE ON my_ht FOR EACH STATEMENT EXECUTE PROCEDURE _timescaledb_internal.on_unsupported_main_table()
+
+-- Should error when adding again
+\set ON_ERROR_STOP 0
+postgres=# ALTER TABLE my_ht ADD COLUMN val2 integer;
```

Comment:
This command and error is incorrect should be `ALTER TABLE my_ht ADD COLUMN val2 integer;` without postgres=#